### PR TITLE
some formatting

### PR DESCRIPTION
**Pull request by OskarStark**

### CODE REVIEW

File: Resources/views/CRUD/base_history.html.twig [hunk 1]
```diff
@@ -27,25 +27,27 @@ file that was distributed with this source code.
...
+                            <th>{{ 'td_compare'|trans({}, 'SonataAdminBundle') }}</th>
                         </tr>
                         </thead>
                         <tbody>
                         {% for revision in revisions %}
                             <tr class="{% if (currentRevision != false and revision.rev == currentRevision.rev) %}current-revision{% endif %}">
-                                <td>{{ revision.rev}}</td>
+                                <td>{{ revision.rev }}</td>
                                 <td>{% include admin.getTemplate('history_revision_timestamp') %}</td>
                                 <td>{{ revision.username}}</td>
```
Comment (qsomazzi) [inline Resources/views/CRUD/base_history.html.twig]: forget one space here ?

Comment (OskarStark) [inline Resources/views/CRUD/base_history.html.twig]: don't understand what you mean

Comment (qsomazzi) [inline Resources/views/CRUD/base_history.html.twig]: you fix ``{{ revision.rev}}`` to ``{{ revision.rev }}`` (l40)
I think you should do it here too : 
``{{ revision. username}}`` to ``{{ revision. username }}`` (l42)

Comment (OskarStark) [inline Resources/views/CRUD/base_history.html.twig]: ah :smile: thank you!

Comment (qsomazzi) [inline Resources/views/CRUD/base_history.html.twig]: :)

Comment (OskarStark) [inline Resources/views/CRUD/base_history.html.twig]: Done, thank you!

**Pull request merged**